gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -678,6 +797,18 @@ def _prepare_device_data(self, device_info):
         device['advance_sharing_interfaces'] = (
                     self._get_advance_sharing_interfaces(device['id']))
         return device
+    
+    def device_healthy(self, event):
+        nfp_context = event.data['nfp_context']
+
+        device = nfp_context['network_function_device']
+
+        self._update_network_function_device_db(device, 'HEALTH_CHECK_COMPLETED')
+
+        self._create_event(event_id='DEVICE_ACTIVE', event_data=nfp_context)
+
+        self._create_event(event_id='CREATE_DEVICE_CONFIGURATION',
+                               event_data=nfp_context, is_internal_event=True)
```

Comment:
race condition - What if generic config is not completed, but stack gets created, in that case traffic might fail.